src/Crocodealer/Config.hs

```diff
@@ -1,18 +1,40 @@
+{- | Configurations through the @config.toml@ file@.
+-}
+
 module Crocodealer.Config
        ( Repo (..)
        , Config (..)
+       , loadConfig
        ) where
 
-import Crocodealer.Core.Label as Label
+import Toml (TomlCodec, (.=))
+import qualified Toml
 
+import Crocodealer.Core.Label as Label
 
 newtype Repo = Repo
     { unRepo :: Text
-    }
+    } deriving (Show)
+
+-- | TOML Codec for the 'Repo' data type.
+repoCodec :: TomlCodec Repo
+repoCodec = Toml.dimap unRepo Repo $ Toml.text "repo"
```

Comment:
You can use `diwrap` codec here, it was specifically created for newtypes :slightly_smiling_face: 
```suggestion
repoCodec = Toml.diwrap $ Toml.text "repo"
```